core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -483,11 +483,14 @@ oppia.directive('conversationSkin', [function() {
             // If the exploration is embedded, use the exploration language
             // as site language. If the exploration language is not supported
             // as site language, English is used as default.
+            var langCodes = $window.GLOBALS.SUPPORTED_SITE_LANGUAGES.map(
+              function(language) {
+                return language.id;
+              });
             if ($scope.isIframed) {
               var explorationLanguageCode = (
                 oppiaPlayerService.getExplorationLanguageCode());
-              if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES[
-                    explorationLanguageCode]) {
+              if (langCodes.indexOf(explorationLanguageCode) > -1) {
```

Comment:
Prefer `if (langCodes.indexOf(explorationLanguageCode) !== -1)`, for consistency across the codebase. This is because there isn't really an order suggested by the `-1`; it's just a symbol that the item is not in the list.